### DISCUSSION TITLE
Error on Kernel Error - ImportError: DLL load failed while importing win32api : The specified procedure could not be found.

### DISCUSSION THREAD

**Discussion by meng032020** [openvinotoolkit/openvino_notebooks#208] [Q&A]
Hi, I have a question about the installation. 2 weeks ago I managed to install the openvino, but today when I reinstall it, I encountered a problem.

The screenshot below shows that the check_install.py is done successfully.
![image](https://user-images.githubusercontent.com/69293028/128977935-e700b7b5-0ec6-4acf-8121-6ef48255982a.png)

However, when I run the jupyter notebook, the error is on kernel dead.
![image](https://user-images.githubusercontent.com/69293028/128978061-4c116e3a-30d8-4086-9748-5887f440c52b.png)
![image](https://user-images.githubusercontent.com/69293028/128978097-bafa5218-ce83-40a6-a379-ac0b939a94b4.png)

I changed the kernel but it is still kernel error.

Additionally, I keep seeing these errors when I run Step 3 (pip install -r requirements.txt):
![image](https://user-images.githubusercontent.com/69293028/128977355-9fbe13aa-77d3-473f-855e-17bd040a73e9.png)

Please kindly advise. thanks!

**Comment by raymondlo84** [comment]
Oh no... 

It looks like the dynamic library got changed... Have you checked if the python is still functional at all? The step is basically doing pip install and can you install other packages at all?

I personally recommend the official python package https://www.python.org/. That would be worth a try as it seems to be more stable on my machine.

**Reply by meng032020** [reply]
Hi Raymond,
Thanks for the reply. It seems that python is still working, although it can be a bit slow. I think I can install the other packages.

**Comment by dhananjayan-aiml** [comment]
Hey, I have converted yolov8 and yolo11 to openvino IR and I have been using it in my project with the device as both CPU /GPU (I'm using DELL Latitude 7490)

    **def get_model(self, model_path: str) -> ov.CompiledModel:
        core = ov.Core()
        model = core.read_model(model_path)
        return core.compile_model(model, device_name="GPU", config={"PERFORMANCE_HINT": "LATENCY"})**

I have switched to iGPU recently as the inference time of my model got reduced from 65ms to 22ms and the inference is fine with no lag, but the issue is I'm getting this **kernal.error.txt** saying:

Instruction / Operand / Region Errors:

/-------------------------------------------!!!KERNEL HEADER ERRORS FOUND!!!-------------------------------------------\
Error in CISA routine with name: kernel
                  Error Message: Input V38 = [256, 260) intersects with V37 = [256, 260)
\----------------------------------------------------------------------------------------------------------------------/


/-------------------------------------------!!!KERNEL HEADER ERRORS FOUND!!!-------------------------------------------\
Error in CISA routine with name: kernel
                  Error Message: Explicit input 2 must not follow an implicit input 0
\----------------------------------------------------------------------------------------------------------------------/

help me understand and resolve it.
